test/cross_origin_test.rb

```diff
@@ -12,7 +12,7 @@ class TestError < StandardError; end
   end
 
   def test_action
-    render :text => "noop"
+    render (::Rails::VERSION::MAJOR < 5 ? :text : :plain) => "noop"
```

Comment:
Is there a better way to do this? `render "noop"` or something so that we don't need to have another switch? If not just do a `head :ok` and test that.